### PR TITLE
Regenerated Network SDK; Fixed customization typos

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Description

PR: ~https://github.com/Azure/azure-rest-api-specs/pull/1726~ https://github.com/Azure/azure-rest-api-specs/pull/1757
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by markcowl — CHANGES_REQUESTED** [review]
Please include the link to the rest spec pr, and the metadata file that is generated when executing generate.cmd

**Comment by EvgenyAgafonchikov** [comment]
@markcowl, I've updated description and metadata. However generate.cmd doesn't work as expected now, because of changed output format of `autorest --list-installed`. Please note this update: https://github.com/Azure/azure-sdk-for-net/pull/3742/commits/f1baba85c699c94d2786dff85ec845f0c3861e06#diff-96cd8d413b9919dc50c4acd59ae4b201. Let me know if you have comments on this change.

File: tools/generateMetadata.ps1
Comment (markcowl) [inline tools/generateMetadata.ps1]: Why is there a change to this file?

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@EvgenyAgafonchikov  The changes look good, except that it appears you updated a file in the tools directory

File: tools/generateMetadata.ps1
Comment (EvgenyAgafonchikov) [reply tools/generateMetadata.ps1]: generate.cmd uses the latest autorest version by default. And currently `autorest --list-installed` has output like this:
```
core       @microsoft.azure/autorest-core                 1.9.3          <Path>
core       @microsoft.azure/autorest-core                 2.0.4143       <Path>
extension  @microsoft.azure/autorest.azureresourceschema  1.9.5          <Path>
extension  @microsoft.azure/autorest.csharp               1.9.3          <Path>
```

Instead of the output previous versions had:
```
AutoRest Code Generator
(c) 2017 Microsoft Corporation. https://aka.ms/autorest
  Build Information
  Autorest Bootstrapper :  1.2.2
  NetCore framework :      1.0.5
  Latest Core Installed :  1.2.2
  Requested Core Version : <none>
Installed versions of AutoRest :

  1.2.2
```

So this line:
```
Write-Host "Latest version:   " (autorest --list-installed | Where {$_ -like "*Latest Core Installed*"}).Split()[-1]
```
causes failure as `Where` part returns empty set of result. This failure is written to metadata in case there is no change in tools/generateMetadata.ps1

**Comment by EvgenyAgafonchikov** [comment]
Please find explanation above.
Temporary tagged as [Do not merge] as there is one more fix in specs that we'd like to get in this version.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@EvgenyAgafonchikov this PR cannot be accepted in it's current state.
I have reached out to you over email explaining you need to merge the latest release first into the public repo and then add the new changes on top of it.
Reopen this PR once you are ready to merge as stated earlier in the email.

**Pull request closed without merging**
